### PR TITLE
link: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #2213

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/link.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# link
+
+> Create a link to an existing file.
```
Comment (sbrl) [inline pages/linux/link.md]: Does it create a symbolic link? If so, then maybe we could reword this to `Create a symbolic link`....?

Also, what about a see also `ln` if more advanced options are required?

Comment (pxgamer) [reply pages/linux/link.md]: It doesn't create a symlink. Only hard links. `ln` seems to be much better as it has symlinks and hard links, as well as error checking.
